releases/unreleased/identity-refresh-bug-for-some-items.yml

```diff
@@ -0,0 +1,9 @@
+---
+title: Identity refresh bug for some items
+category: fixed
+author: Jose Javier Merchante <jjmerchante@bitergia.com>
+issue: null
+notes: >
+  Fix issue where some fields were not refreshed
+  in the identities database under certain
+  circumstances.
```

Comment:
I would be good to explain here that circustances.

Reply:
I updated the changelog, let me know if that is enough or if I should include more information.